src/ripple/test/jtx/JTx.h

```diff
@@ -56,12 +59,99 @@ struct JTx
     {
     }
 
+#if defined(_MSC_VER) && _MSC_VER <= 1800
+    JTx(JTx&& src)
```

Comment:
Since there is a user defined copy constructor, a default compiler will not generate a move constructor. Here we explicitly define a move constructor, but only for visual studio. There probably needs to be al `#else` clause here to define `JTx(JTx&&) = default`;